confdgnmi/src/confd_gnmi_client.py

```diff
@@ -19,6 +19,22 @@
 log = logging.getLogger('confd_gnmi_client')
 
 
+def encoding_to_str(value: int) -> str:
+    """ Convert numeric value of `enum Encoding` to a text
+        according to `gnmi.proto` specification. """
+    if value == 0:
```

Comment:
Why not use `gnmi_pb2.` values here? like  `gnmi_pb2.Encoding.JSON`  Can we be even sure numbers a guaranteed in future versions of gnmi.proto (perhaps yes)?

Reply:
I don't think future versions will stop using numbers, but still, this comment is valid.

Reply:
replaced with enum values - wanted to reduce dependency on low level lib usage by using the proto/standard values